src/openms/include/OpenMS/ANALYSIS/ID/PeptideProteinResolution.h

```diff
@@ -93,7 +93,9 @@ namespace OpenMS
     // We always take first PepHit from PepID, because those were used for
     // inference in Fido
     typedef std::map<Size, std::set<Size> > IndexMap_;
-    
+
+    /// if the protein group at index i contains a target (first) and/or decoy (second)
+    // std::vector<std::pair<bool,bool>> indist_prot_grp_td_;
```

Comment:
remove comment?

Reply:
hmm jaa.. this is part of a WIP method. I would like to keep it for now. I don't know how to stash it best. I promise to remove or implement. Maybe I switch to resolving on the Boost graph anyway.

Reply:
I commented it though.